README.md

```diff
@@ -253,6 +253,11 @@ The next segments have (sub)settings of their own:
     directory is shortened to `TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS_LENGTH`.
     - `TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS_LENGTH=1`: length of each parent
         directory when `TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS` is enabled.
+- prompt:
+    - `TRUELINE_SHOW_TRAILING_SEPARATOR=true`: boolean variable that determines 
```

Comment:
Move this to the `external` subsection (just before the `TRUELINE_SHOW_VIMODE` and remove the `prompt` bullet (i.e simply add the option).